### ISSUE TITLE
GUI not accessible for screen readers

### ISSUE THREAD

**Issue by tommyTsaiTou** [ihaveamac/custom-install#61]
The tkinter GUI is not accessibile for screen readers can you please make a wx one?
So blind or visually impaired user can use this application easily.
thanks

**Comment by ihaveamac** [comment]
I'm sorry that the gui is inaccessible to screen readers. I will look into trying out wxpython, though I can't guarantee any kind of time for that.

**Comment by tommyTsaiTou** [comment]
Don't forget button's label and name, they are the most important part for screen readers to read an object.

**Comment by tommyTsaiTou** [comment]
For now chat gpt is so powerful, you can just submit the tkinter's object code and ask it to convert it into wxpython code for you and it will work.
